crates/invoker-impl/src/invocation_task/service_protocol_runner.rs

```diff
@@ -126,11 +126,7 @@ where
         let journal_size = journal_metadata.length;
 
         // Attach parent and uri to the current span
-        let invocation_task_span = Span::current();
-        journal_metadata
-            .span_context
-            .as_parent()
-            .attach_to_span(&invocation_task_span);
+        Span::current().set_relation(journal_metadata.span_context.as_parent());
```

Comment:
I am wondering whether these internal spans should really be associated with the services trace. Maybe it is simplest if we keep the services and runtime traces separate because then there is no problem if we send them to two different jaeger endpoints.